tools/parse_baselines/main.py

```diff
@@ -2,28 +2,31 @@
 # SPDX-License-Identifier: Apache-2.0
 """Script used to calculate baselines from raw performance test output."""
 
+# pylint: disable=wrong-import-position
```

Comment:
maybe add a comment about why this is required

Reply:
Done.